src/NServiceBus.AcceptanceTests/Mutators/When_using_outgoing_tm_mutator.cs

```diff
@@ -39,7 +39,6 @@ public Task MutateOutgoing(MutateOutgoingTransportMessageContext context)
                 {
                     context.OutgoingHeaders["HeaderSetByMutator"] = "some value";
                     context.OutgoingHeaders[Headers.EnclosedMessageTypes] = typeof(MessageThatMutatorChangesTo).FullName;
-                    context.OutgoingBody = Encoding.UTF8.GetBytes("<MessageThatMutatorChangesTo/>");
```

Comment:
I still think you have to set this.  How else would https://github.com/Particular/NServiceBus/blob/RemoveXmlFromAcceptanceTests/src/NServiceBus.AcceptanceTests/Mutators/When_using_outgoing_tm_mutator.cs#L60 be invoked? 

How about we change the test to set the serializer to XML explicitly, that way it should work for all transports?